### PR TITLE
Fix issue in getting started tutorial using wikipedia data

### PR DESCRIPTION
**Pull request by RohitRathore1**

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -48,7 +48,6 @@ We use [this script](https://github.com/allenai/dolma/blob/main/scripts/make_wik
 ```shell
 python scripts/make_wikipedia.py \
   --output wikipedia \
-  --languages simple \
```
Comment (soldni) [inline docs/getting-started.md:51]: Why changing this line?

**Review by soldni — COMMENTED** [review]
Hello! Thank you for your PR. Left a comment to explain changes. A better title for this PR would also help.

File: docs/getting-started.md (hunk 1 above)
Comment (RohitRathore1) [reply docs/getting-started.md:51]: hi, we had already provided `--lang simple`

**Comment by RohitRathore1** [comment]
> Hello! Thank you for your PR. Left a comment to explain the changes. A better title for this PR would also help.

Hi, sorry for the delay in reply. I had missed this.

**Review by soldni — APPROVED** [review]

**Pull request merged**